interface/patient_file/summary/demographics.php

```diff
@@ -1844,6 +1844,12 @@ function setMyPatient() {
             ];
             checkSkipConditions();
 
+            <?php
```

Comment:
Since the event is now more content agnostic, this event needs to fire at the outermost part of the page DOM. I would move it after the closing `</body>` tag. If it is kept here, a user could use the event to inject CSS or HTML but it would be rendered inside of a `<script>` tag (which starts at line 1840).

Also, line 72 creates `$ed` which is a helper variable, pointing to `$GLOBALS['kernel']->getEventDispatcher()` so you could consolidate this to just be `$ed->dispatch(new RenderEvent($pid), RenderEvent::EVENT_RENDER_POST_PAGELOAD, 10);` though it is not necessary.